google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/EndpointTestBase.java

```diff
@@ -51,10 +51,6 @@ protected void tearDown() throws Exception {
     }
     finally {
       super.tearDown();
-
-      if (myFixture != null && getProject() != null && !getProject().isDisposed()) {
-        Disposer.dispose(getProject());
```

Comment:
on that note, we once were failing builds on unused imports. worth looking into why this isn't happening anymore.